extensions/amp-list/0.1/amp-list.css

```diff
@@ -51,6 +51,15 @@ amp-list[load-more] [load-more-button].i-amphtml-default-ui,
   bottom: 0;
 }
 
+[load-more] div[role="list"].i-amphtml-list-load-more-container-overflow {
```

Comment:
what makes the overflow case different from the other default UI cases? id est, why do we have this block of the CSS when the others are styled with the selectors on line 25, for example?

Reply:
This is to override CSS applied by the `i-amphtml-fill-content i-amphtml-replaced-content` classes in order to allow us to correctly apply a `margin-bottom`.

Reply:
This is now deleted because we removed `applyFillContent` for the `load-more` use case.